src/frontend/src/pages/DesignReviewEditModal.tsx

```diff
@@ -0,0 +1,92 @@
+import { Grid } from '@mui/material';
+import NERModal from '../components/NERModal';
+import { DRCModalProps, times, daysOfWeek, TimeSlot } from './DesignReviewCommon';
+import { useState } from 'react';
+
+const DRCEditModal: React.FC<DRCModalProps> = ({ open, onHide, onSubmit, title, iconData }) => {
+  const header = `Are you availble for the ${title} Design Review at 9:00 in the Bay`;
```

Comment:
the 'in the Bay' part should be a prop that someone can pass in, because DRs also have other locations/ can be on zoom